docs/build.md

```diff
@@ -2,40 +2,107 @@
 
 Dependency | Version | Required           | Runtime
 ---        | ---     | ---                | ---
-Kokkos     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
+[Kokkos](https://github.com/kokkos/kokkos)     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
 MPI        | 2       |                    | :heavy_check_mark:
 CMake      | 3.12    | :heavy_check_mark: |
 
-# Installation
+# Build and installation
 
-## Installation using Spack
+The build instructions for the Kokkos library can be found
+[here](https://github.com/kokkos/kokkos#building-and-installing-kokkos).
+ArborX requires the following options to be set in the Kokkos CMake build:
+- `Kokkos_CXX_STANDARD=14`
```

Comment:
Not true

Reply:
Dude, it's such a job to pull out words out of your mouth :( Please elaborate.

Reply:
Is it simply `CMAKE_CXX_STANDARD=14` or later?

Reply:
No you just don't need to specify anything.  You can build ArborX against a Kokkos that was built with C++11 just fine.

Reply:
Huh.. I guess I always thought we needed that, and in our [current](https://github.com/arborx/ArborX/blob/master/docs/build.md#L27) documentation we also specify that (though only for Trilinos). I'll remove it then.